.eslintrc

```diff
@@ -31,11 +31,12 @@
         "SwitchCase": 1
       }
     ],
-    "import/no-extraneous-dependencies": [error, { "devDependencies": true }],
+    "import/no-extraneous-dependencies": ["error", { "devDependencies": true }],
     "import/namespace": 0,
     "import/named": 0,
     "import/prefer-default-export": 0,
     "import/default": 0,
+    "import/resolver": { "node": { "extensions": [".js",".jsx"] } },
```

Comment:
According to my IDE this config for import/resolver is invalid. It's expecting 0, 1 or 2.

Reply:
should be fixed in [170a9c1](https://github.com/auth0/auth0-authorization-extension/pull/387/commits/170a9c1d068bd9e5a13e2650b4b92b17a4aaf255) 😄